lambda/rotator-snowflake/rotate/snowflake.go

```diff
@@ -4,12 +4,14 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
+	"strings"
 
 	"github.com/chanzuckerberg/go-misc/snowflake"
 )
 
 func updateSnowflake(ctx context.Context, user string, db *sql.DB, pubKey string) error {
-	query := fmt.Sprintf(`ALTER USER "%s" SET RSA_PUBLIC_KEY_2 = "%s"`, user, pubKey)
+	cappedUser := strings.ToUpper(user)
+	query := fmt.Sprintf(`ALTER USER "%s" SET RSA_PUBLIC_KEY_2 = "%s"`, cappedUser, pubKey)
```

Comment:
We still need to send the username in all caps because Snowflake SCIM capitalizes all the users.